Moto/G8Plus/res/values/config.xml

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="config_dozeAfterScreenOff">false</bool>
+    <bool name="config_dynamic_bind_ims">true</bool>
+    <string name="config_ims_package">org.codeaurora.ims</string>
+    <bool name="config_powerDecoupleInteractiveModeFromDisplay">true</bool>
+    <bool name="config_setColorTransformAccelerated">true</bool>
+    <bool name="config_pinnerCameraApp">true</bool>
+    <bool name="config_useDevInputEventForAudioJack">true</bool>
+    <integer name="config_bluetooth_operating_voltage_mv">3300</integer>
+    <integer name="config_screenBrightnessDoze">17</integer>
+    <bool name="config_automatic_brightness_available">true</bool>
+    <bool name="config_bluetooth_hfp_inband_ringing_support">true</bool>
+    <bool name="config_bluetooth_le_peripheral_mode_supported">true</bool>
+    <bool name="config_dozeAfterScreenOffByDefault">false</bool>
+    <bool name="config_dozeAlwaysOnDisplayAvailable">true</bool>
+    <bool name="config_carrier_volte_available">true</bool>
+    <bool name="config_device_volte_available">true</bool>
+    <bool name="config_device_vt_available">true</bool>
+    <bool name="config_device_wfc_ims_available">true</bool>
+    <bool name="config_carrier_wfc_ims_available">true</bool>
+
+    <bool name="config_hotswapCapable">true</bool>
+
+    <bool name="config_nightDisplayAvailable">true</bool>
```

Comment:
AOSP has
`<bool name="config_nightDisplayAvailable">@bool/config_setColorTransformAccelerated</bool>`
So this is useless